data/tlg0007/tlg071/tlg0007.tlg071.perseus-eng4.xml

```diff
@@ -0,0 +1,1125 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?>
+<!--<!DOCTYPE TEI.2
+  PUBLIC "-//TEI P4//DTD Main DTD Driver File//EN" "http://www.tei-c.org/Guidelines/DTD/tei2.dtd" [
+<!ENTITY % TEI.XML "INCLUDE">
+<!ENTITY % PersProse PUBLIC "-//Perseus P4//DTD Perseus Prose//EN" "http://www.perseus.tufts.edu/DTD/1.0/PersProse.dtd" >
+%PersProse;
+]>-->
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+   <teiHeader n="CTS">
+      <fileDesc>
+         <titleStmt>
+            <title xml:lang="lat">How a Man May Be Sensible of His Progress in Virtue</title>
+            <!--<title type="sub">Machine readable text</title>-->
+            <author>Plutarch</author>
+            <editor role="editor">Goodwin</editor>
```

Comment:
please add his full name `<editor role="editor">William W. Goodwin</editor>` and add
`<editor role="translator">Hugh Tod(d)</editor>`